cime_config/acme/machines/config_compilers.xml

```diff
@@ -590,6 +590,24 @@ for mct, etc.
   <MPICXX> CC </MPICXX>
 </compiler>
 
+<compiler COMPILER="intel" MACH="theta">
```

Comment:
The use of `-xMIC-AVX512` in compile flags is new and is needed for KNLs. Also, `-openmp` will cause lots of `deprecated` warnings in builds logs: this flag is now `-qopenmp` starting with Intel v17 compiler.